app/src/main/java/kr/co/connect/boostcamp/livewhere/util/ConstVal.kt

```diff
@@ -0,0 +1,34 @@
+package kr.co.connect.boostcamp.livewhere.util
+
+// FIXME 상수이름은 camelCase나 snake_case보다는 CONSTANT_VARIABLE_NAME과 같은 이름의 패턴으로 사용해주세요.
+const val AVG_PRICE = "연도별 평균 거래가"
+const val RADIUS = 500
+const val EMPTY_BARCHART_TEXT = "거래 기록이 없습니다"
+const val EMPTY_REVIEW_TEXT = "해당 지역의 후기가 없습니다."
+const val TYPE_CHARTER = 1
+const val TYPE_MONTHLY = 2
+const val SORT_BY_AREA = 1
+const val SORT_BY_AREA_REV = -1
+const val SORT_BY_TYPE = 2
+const val SORT_BY_TYPE_REV = -2
+const val SORT_BY_YEAR = 3
+const val SORT_BY_YEAR_REV = -3
+const val TIME_LIMIT = 2000
```

Comment:
```suggestion
const val TIME_LIMIT = 2_000
```
P3. 천단위에서 언더바를 사용하시면 가독성이 좋아집니다